### ISSUE TITLE
Server Error (500) when accessing /admin or /accounts

### ISSUE THREAD

**Issue by Tzzui** [mediacms-io/mediacms#1004]
**Describe the issue**
Trying to access https://URL/accounts/login/ or https://URL/admin only returns a "Server Error (500)"

**To Reproduce**
Steps to reproduce the issue:
1. Go to https://URL/accounts/login/
2. See error

Other site like https://URL/view?m=luJc6Jfmd or https://URL/members work fine 

**Expected behavior**
I expect to see the according site

**Screenshots**
![grafik](https://github.com/mediacms-io/mediacms/assets/71082590/b834d94d-cce9-42bf-ba88-786e34559a88)

**Environment (please complete the following information):**
 - OS: Ubuntu Linux
 - Installation method: Docker install
 - any browser

**Additional context**

Docker logs

mediacms-web-1
192.168.128.3 - - [19/Apr/2024:12:07:32 +0000] "GET /accounts/login/ HTTP/1.1" 500 145 "https://URL/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0"
[pid: 62|app: 0|req: 1384/2865] 192.168.128.3 () {66 vars in 1164 bytes} [Fri Apr 19 13:07:32 2024] GET /accounts/login/ => generated 145 bytes in 16 msecs (HTTP/1.1 500) 7 headers in 244 bytes (1 switches on core 0)

nginx-proxy
nginx.1     | URL 10.20.8.1 - - [19/Apr/2024:12:09:16 +0000] "GET /accounts/login/ HTTP/2.0" 500 145 "https://URL/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0" "192.168.128.8:80"

**Comment by tiritibambix** [comment]
I got the same error.
I'm using default `settings.py`. Is there something to be changed in there ?

**Comment by tiritibambix** [comment]
>Tzzui closed this as completed

Is it though ?
[docker-compose-named-volumes.yaml](https://github.com/mediacms-io/mediacms/blob/main/docker-compose-named-volumes.yaml) doesn't mention volumes like in the solution I found and the docs lack clarity about setting `ENABLE_UWSGI` to `yes` for the login / register page to work.

The struggle was real to get MediaCMS to work. I wish I knew how to code and how to work on github to change things. Only thing I have to offer is the solution I linked above. But I really believe docker support could be reworked.
